modules/manage/pages/schema-registry/schema-reg-ui.adoc

```diff
@@ -0,0 +1,116 @@
+
+=== Topic strategy use case
+
+The Topic strategy is suitable when you want to group schemas by the topics to which they are associated. Suppose you're tracking product order information in a topic named Transactions. When a producer sends records to the OrderInfo topic, you want the record names to look something like: 
+
+- Transactions - Record1 
+- Transactions - Record2 
+
+Where `Record1` and `Record2` are unique identifiers. This is usually defined in your producer settings. Create your schema with the Topic strategy, and the subject name is always Transactions, with all customer transactions under the same topic. 
+
+=== Record strategy use case
+
+The Record strategy is most useful when you have multiple schemas within a topic and need more granular categorization that's influenced by the record name. Suppose there's an Events topic with event types A and B. You may want each of those event types to have their own subject, their own schemas, and their own fully-qualified record names (for example, com.example.EventTypeA). If each event type has its own schema with the Record strategy, then when producers send these event types to the Events topic, their subjects are those record names: 
+
+- com.example.EventTypeA
+- com.example.EventTypeB
+
+The record names in the Events topic look like this: 
+
+- Events-com.example.EventTypeA-Record1
+- Events-com.example.EventTypeB-Record1
+- Events-com.example.EventTypeA-Record2
+- Events-com.example.EventTypeB-Record2
+
+=== TopicRecord strategy use case
+
+The TopicRecord strategy is suitable when you want to organize schemas based on both topics and logical record types. Suppose there's a microservices architecture where different services produce to the same topic: SharedEvents. Each microservice has a schema of its own for the shared events, but each schema uses the TopicRecord strategy. This results in the following subject names: 
+
+- SharedEvents-com.example.MicroserviceAEvent
+- SharedEvents-com.example.MicroserviceBEvent
+
+The record names look like this: 
+
+- SharedEvents-com.example.MicroserviceAEvent-Record1
+- SharedEvents-com.example.MicroserviceBEvent-Record1
+- SharedEvents-com.example.MicroserviceAEvent-Record2
+- SharedEvents-com.example.MicroserviceBEvent-Record2
+
+This allows for multiple schemas to govern the same shared events for different microservices, allowing granular organization. 
+
+== Configure schema compatibility
+
+include::partial$schema-compatibility.adoc[]
+
+== Delete a schema
+
+Select a schema to soft-delete its entire subject or a single selected version of it. A soft-deleted schema can be recovered, but a permanently-deleted schema cannot be recovered. Schemas cannot be permanently deleted if any other schemas reference it. They remain readable with a soft delete until you remove all references to it. Redpanda does not recommend permanently deleting schemas in a production environment.
```

Comment:
> Schemas cannot be permanently deleted if any other schemas reference it. They remain readable with a soft delete until you remove all references to it.

It’s not possible to soft-delete schemas that are still being referenced afaik. We can test this or ask @BenPope to verify

Reply:
This is a bit nit-picky, but:

> Schemas cannot be permanently deleted if any other schemas reference it

Schemas cannot be deleted (by schema id), only subjects or subject-versions.

> They remain readable with a soft delete until you remove all references to it

They're still readable until it's permanently deleted.

> It’s not possible to soft-delete schemas that are still being referenced afaik.

Subject-versions cannot be deleted if they are referenced.

Reply:
Does deletion work differently in the UI and the API? With API, we say [here](https://deploy-preview-86--redpanda-docs-preview.netlify.app/current/manage/schema-reg/schema-reg-api/#delete-a-schema) that you can delete a single schema or all schemas of a subject. @BenPope or @weeco: Is it correct to leave that API page as is, and change the UI page [Delete a schema](https://deploy-preview-86--redpanda-docs-preview.netlify.app/current/manage/schema-reg/schema-reg-ui/#delete-a-schema) to: 

**Delete a subject**
In the UI, you can delete a subject or a version of a subject. You cannot delete individual schemas. Subjects remain readable with a soft delete until you remove all references to it. A soft-deleted subject can be recovered, but a permanently-deleted subject cannot be recovered. Redpanda does not recommend permanently deleting subjects in a production environment.